events/renderers/docx.py

```diff
@@ -0,0 +1,269 @@
+import io
+from collections import OrderedDict
+
+import dateutil
+from bs4 import BeautifulSoup
+from django.utils.text import slugify
+from django.utils.timezone import localtime
+from docx import Document
+from rest_framework import renderers
+from rest_framework.exceptions import APIException
+
+
+def get_any_language(dictionary, default='fi', language_codes=None):
+    if not language_codes:
+        # Default order for when a language isn't found.
+        language_codes = list([default, 'fi', 'sv', 'en'])
+
+    for language_code in language_codes:
+        content = dictionary.get(language_code)
+        if content:
+            return content
+    return ''
+
+
+def get_description(event):
+    if event['short_description']:
+        description_with_html = event['short_description']
+    elif event['description']:
+        description_with_html = event['description']
+    return description_with_html
```

Comment:
This will fail with an `UnboundLocalError` if the event has neither short description or description, but using `return` as I advise above will fix that.